CRM/Event/ActionMapping.php

```diff
@@ -158,7 +158,7 @@ public function createQuery($schedule, $phase, $defaultParams) {
     $query['casContactTableAlias'] = NULL;
     $query['casDateField'] = str_replace('event_', 'r.', $schedule->start_action_date);
     if (empty($query['casDateField']) && $schedule->absolute_date) {
-      $query['casDateField'] = $schedule->absolute_date;
+      $query['casDateField'] = "'" . CRM_Utils_Type::escape($schedule->absolute_date, 'String') . "'";
```

Comment:
Could we parameterize it?
$query['casDateField'] = '@absoluteDate'
$query->param('absoluteDate', $schedule->absolute_date);

Reply:
I like that better, but unfortunately it doesn't work because the casDateField gets put put into the value for !repetitionEndDate and that value doesn't get interpolated so it just ends up being @absoluteValue in the final SQL.

Reply:
I see.  A slight improvement could be:
$query['casDateField'] = $query->escapeString($schedule->absolute_date);